### PR TITLE
add new terms mild ichthyosis vulgaris and severe ichthyosis vulgaris

### LINKED ISSUE #2360: [NTR] Subtypes in MONDO
Open question:
Should we have two subtypes of Ichthyosis vulgaris, corresponding to a mild and severe form. We may need to bring Courtney into this discussion.

_Update 2022-11-30 by @nicolevasilevsky _

---

Ichthyosis vulgaris (https://omim.org/entry/146700) has currently one entry in MONDO (http://purl.obolibrary.org/obo/MONDO_0024304). However, heterozygote FLG mutation carriers have mild manifestations, while homozygous mutation carriers have more severe manifestations. For computational analysis, it would be good to have separate entries. Ideally this would be done together with OMIM. @ahamosh 
Can we discuss a strategy for this? There are many analogous examples.

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #2360

### CODE REVIEW

File: src/patterns/dosdp-patterns/mild.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+pattern_name: mild
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/mild.yaml
+
+description: 'Pattern for describing the severity of a disease, in this case, a mild form of the disease.  
+Here mild means means having a relatively minor degree of severity. This may correspond with specific genetic mutations (or homozygous or heterozygous forms).
+
```
Comment (sabrinatoro) [inline src/patterns/dosdp-patterns/mild.yaml:7]: typo: remove one "means".

File: src/patterns/dosdp-patterns/severe.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+pattern_name: severe
+
+pattern_iri: http://purl.obolibrary.org/obo/mondo/patterns/severe.yaml
+
+description: 'Pattern for describing the severity of a disease, in this case, a severe form of the disease.  
+Here severe means means having a high degree of severity. This may correspond with specific genetic mutations (or homozygous or heterozygous forms).
+
```
Comment (sabrinatoro) [inline src/patterns/dosdp-patterns/severe.yaml:7]: typo: remove one "means"

File: src/patterns/dosdp-patterns/severe.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+vars:
+  disease: '''disease'''
+
+name:
+  text: severe %s
+  vars:
+  - disease
+
+def:
+  text: An instance of %s in which the disease presentation is severe in severity..
```
Comment (sabrinatoro) [inline src/patterns/dosdp-patterns/severe.yaml:33]: typo: remove one of the periods at the end

**Review by sabrinatoro — APPROVED** [review]
@nicolevasilevsky 
It looks good to me. Only a few minor typos to fix.

File: src/patterns/dosdp-patterns/severe.yaml (hunk 3 above)
Comment (nicolevasilevsky) [reply src/patterns/dosdp-patterns/severe.yaml]: thank you!

**Pull request merged**